packages/shared/src/consts/topicKeysRegex.ts

```diff
@@ -0,0 +1 @@
+export const TOPIC_KEYS_REGEX = /^(?:[a-zA-Z0-9_-]+|\S+@\S+\.\S+)$/;
```

Comment:
Similarly to the subscriber story, I suggest using two separate regexes, one for the ASCI stuff and another for the email regex.

Also please add all of these regexes for topics and subscribers to a single const file. That is combine `subscriberIdRegex` and `topicKeysRegex`.